lib/request/TingClientSearchRequest.php

```diff
@@ -212,6 +212,7 @@ public function processResponse(stdClass $response) {
     $searchResult->numTotalObjects = self::getValue($searchResponse->result->hitCount);
     $searchResult->numTotalCollections = self::getValue($searchResponse->result->collectionCount);
     $searchResult->more = (strcasecmp('true', self::getValue($searchResponse->result->more)) == 0);
+    $searchResult->sortUsed = self::getValue($searchResponse->result->sortUsed);
```

Comment:
This sets a property on the object which does not seem to be defined in the class. I do not consider that good practice but it seems to be the current course of things here anyway.